styx-scheduler-service/src/test/java/com/spotify/styx/docker/KubernetesPodEventTranslatorTest.java

```diff
@@ -130,6 +130,16 @@ public void runErrorOnUnexpectedTerminatedStatus() throws Exception {
   }
 
   @Test
+  public void runErrorOnLostNode() throws Exception {
+    pod.setStatus(running(true));
+    pod.getStatus().setReason("NodeLost");
+
+    assertGeneratesEventsAndTransitions(
+        RunState.State.RUNNING, pod,
+        Event.runError(WFI, "Node became unresponsive"));
```

Comment:
The code seems to return `Kubernetes node became unresponsive` and not `Node became unresponsive`?